package.json

```diff
@@ -75,7 +76,7 @@
     "eslint-plugin-qunit": "^6.2.0",
     "gfm-code-blocks": "^1.0.0",
     "guidemaker": "^2.4.0",
-    "guidemaker-default-template": "^5.0.0",
+    "guidemaker-ember-locale-template": "git://github.com/DazzlingFugu/guidemaker-ember-locale-template#main",
```

Comment:
Is mentioning `#main` necessary? I don't remember very well.

Reply:
I don't remember either, I have always put the name of the branch to remember what I am doing, that's a habit ^^